src/cli.ts

```diff
@@ -15,16 +15,17 @@ const RESOURCES = "/resources";
 const frontendAssets = RESOURCES;
 const coreResources = `${RESOURCES}/Resources.php`;
 
-if (process.argv.length === 3) {
+if (process.argv.length >= 3) {
```

Comment:
Could you constrain this to be either 3 or 4 only? Anything else is not really valid and should show the error message in the else

Reply:
Also, if you could update the usage error in the else to tell the user about the mandatory and optional parameter it would be great 😊

Reply:
Done in https://github.com/wikimedia/resource-modules/pull/3/commits/6c8be477d8428cd9c2789a153a26711d65314d1d!